web/src/OverviewTableTriggerButton.tsx

```diff
@@ -64,80 +55,48 @@ type TriggerButtonProps = {
   isQueued: boolean
   resourceName: string
   analyticsTags: Tags
-}
-
-const titleText = (
-  isDisabled: boolean,
-  shouldbeClicked: boolean,
-  isQueued: boolean
-): string => {
-  if (isQueued) {
-    return TriggerButtonTooltip.AlreadyQueued
-  } else if (isDisabled) {
-    return TriggerButtonTooltip.UpdateInProgOrPending
-  } else if (shouldbeClicked) {
-    return TriggerButtonTooltip.NeedsManualTrigger
-  } else {
-    return TriggerButtonTooltip.Default
-  }
+  onTrigger: () => void
 }
 
 function OverviewTableTriggerButton(props: TriggerButtonProps) {
-  let isManualTriggerMode = props.triggerMode !== TriggerMode.TriggerModeAuto
-  let isManualTriggerIncludingInitial =
-    props.triggerMode === TriggerMode.TriggerModeManual
+  let isManual =
+    props.triggerMode === TriggerMode.TriggerModeManual ||
+    props.triggerMode === TriggerMode.TriggerModeManualWithAutoInit
+  let isAutoInit =
+    props.triggerMode === TriggerMode.TriggerModeAuto ||
+    props.triggerMode === TriggerMode.TriggerModeManualWithAutoInit
 
-  // trigger button will only look actionable if there isn't any pending / active build
-  let isDisabled =
-    props.isQueued || // already queued for manual run
-    props.isBuilding || // currently building
-    (!isManualTriggerIncludingInitial && !props.hasBuilt) // waiting to perform its initial build
+  // clickable (i.e. trigger button will appear) if it doesn't already have some kind of pending / active build
+  let clickable =
+    !props.isQueued && // already queued for manual run
+    !props.isBuilding && // currently building
+    !(isAutoInit && !props.hasBuilt) // waiting to perform its initial build
 
-  let shouldBeClicked = false
-  if (!isDisabled) {
-    if (props.hasPendingChanges && isManualTriggerMode) {
-      shouldBeClicked = true
-    } else if (!props.hasBuilt && isManualTriggerIncludingInitial) {
-      shouldBeClicked = true
+  let isBold = false
+  if (clickable) {
+    if (props.hasPendingChanges && isManual) {
+      isBold = true
+    } else if (!props.hasBuilt && !isAutoInit) {
+      isBold = true
```

Comment:
fyi, there is a slight discrepancy between table view and detail view when a manual resource has been triggered once, even though it looks like this logic is the same! i did check on the main branch and it seems like this is happening there too, so it's not really anything to fix here.

detail view fills the icon with blue:
![Screen Shot 2021-12-08 at 11 58 09 AM](https://user-images.githubusercontent.com/23283986/145251121-ea2b7093-8622-40ab-9282-06933840eb23.png)

table view does not (but it is blue when the resource is manual init and has not been built):
![Screen Shot 2021-12-08 at 11 58 02 AM](https://user-images.githubusercontent.com/23283986/145251128-1d809960-acc4-459c-99fe-737feaa83c9f.png)

Reply:
haha -- yahhhhh, there are a bunch of discrepancies here. honestly i meant this to be a 2-line PR but it was hard to resist not fixing some of them (tho there are still some left)